src/sider/historikk/Historikk.tsx

```diff
@@ -129,20 +132,31 @@ export function Historikk({ historikkEvents, tilfeller }: Props): ReactElement {
 
   return (
     <Box background="surface-default" padding="4">
-      <Select
-        className="w-fit mb-4"
-        label={texts.sykefravaerstilfelleLabel}
-        onChange={sykefravaerstilfelleOnChange}
-      >
-        {tilfeller.map((tilfelle, index) => (
-          <option key={index} value={index}>
-            {tilLesbarPeriodeMedArstall(tilfelle.start, tilfelle.end)}
-          </option>
-        ))}
-        {eventUtenforTilfelleList.length > 0 && (
-          <option value={-1}>Utenfor sykefraværstilfelle</option>
-        )}
-      </Select>
+      <HStack justify="space-between" align="center">
+        <Select
+          className="w-fit mb-4"
+          label={texts.sykefravaerstilfelleLabel}
+          onChange={sykefravaerstilfelleOnChange}
+        >
+          {tilfeller.map((tilfelle, index) => (
+            <option key={index} value={index}>
+              {tilLesbarPeriodeMedArstall(tilfelle.start, tilfelle.end)}
+            </option>
+          ))}
+          {eventUtenforTilfelleList.length > 0 && (
+            <option value={-1}>Utenfor sykefraværstilfelle</option>
+          )}
+        </Select>
+        <EksternLenke
+          href={fullNaisUrlIntern(
+            "veilarbpersonflate",
+            "/arbeidsmarkedstiltak"
+          )}
```

Comment:
Denne funksjonen tar seg av dev vs prod for lenkene 👍🏼

Reply:
Håndterer `.ansatt` og regner jeg med?

Reply:
Japp 👍🏼